book/website/ethical-research/activism/activism-case-study-google.md

```diff
@@ -0,0 +1,53 @@
+(er-activism-case-study-google)=
+# Case study: Google Workers & Project Maven
+
+Tech workers have historically had low levels of unionisation for a variety of different reasons. 
+Some have also been ambivalent about the potential impact of the companies and products they work for on systemic inequality and injustice across the globe.
+
+However, more and more people who work in the industry are speaking up and demanding that companies and organisations consider the ethics and impact of the work they do and who they do that work with. 
+
+Google is one example of a large tech company where workers have spoken up on several different issues including internal company policy and external contracts. 
+
+(er-activism-case-study-google-advocacy)=
+## Advocacy & Workplace Activism by Google Workers 
+### Project Maven - 2018
+One example of this is Google workers campaigning against Project Maven, a contract with the US Pentagon to use AI to develop more effective drone guidance.
+Google employees across the globe spoke up in internal meetings, signed petitions, made memes, and shared personal statements about why they disagreed with the contract.
+Some employees even resigned.
+Eventually, the leadership at Google agreed not to seek further contracts with the US Pentagon and also agreed to adopt ["Ethical AI Principles"](https://blog.google/technology/ai/ai-principles/) including not using its technology for weapons or surveillance.
+[Source, accessed 8 December 2020](https://jacobinmag.com/2018/06/google-project-maven-military-tech-workers)
+
+### Walkout for Real Change - 2018 
+In response to allegations of million dollar exit packages for executives accused of sexual harassment against workers, workers at Google offices across the globe walked out demanding: 
+* Ending of private arbitration in sexual harassment cases 
+* Publication of a transparency report on instances of sexual harassment
+* Further disclosures of salaries and compensation
+* An employee representative on the company board
+* A chief diversity officer who could speak directly to the board
+
+Google agreed to end the use of private arbitration in sexual harassment cases. 
```

Comment:
Did they agree to any of the other demands?

Reply:
Nope :(